src/services/compare-appian/handlers/sendNoMatchAlert.ts

```diff
@@ -21,80 +37,79 @@ exports.handler = async function (
 
   if (!region) throw "process.env.region needs to be defined.";
 
-  // use this secret path to define the { emailRecipients, sourceEmail } for the does not match email
   const secretId = `${project}/${stage}/alerts-appian`;
 
-  const data = { ...event.Payload };
+  const data = { ...event.Payload } as Types.AppianSeatoolCompareData;
   const id: string = data.SPA_ID;
+  const secretExists = await Libs.doesSecretExist(region, secretId);
+  const secSinceAppianSubmitted = data.secSinceAppianSubmitted || 0;
 
-  const secretExists = await doesSecretExist(region, secretId);
+  // Was this submitted more than five days ago? If so, it's urgent:
+  const isUrgent = secSinceAppianSubmitted >= 432000; // Five days in secs
```

Comment:
Was this a new business requirement from Jonya?

Reply:
It appeared to be, judging from the email templates. But I'm not sure. That's a great question, I'll talk to Dany to get confirmation from her.